hasura/graphql-engine
# Add API docs for `webhook_from_env`

### Description
Add documentation to create a trigger that targets an environment based webhook via API.
fixes #4419

### Changelog

- [x] `docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst` is updated with a new property in create triggers.

### Affected components


### Related Issues
#4419


### Steps to test and verify
Following this change in the doc, the tester should be able to use Hasura API to create a new trigger who's webhook URL is defined as an environment variable.

### Limitations, known bugs & workarounds



### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?



#### GraphQL

#### Breaking changes
